site/src/pages/DeploymentSettingsPage/GeneralSettingsPage/GeneralSettingsPageView.tsx

```diff
@@ -54,6 +61,32 @@ export const GeneralSettingsPageView: FC<GeneralSettingsPageViewProps> = ({
 						</ChartSection>
 					</div>
 				)}
+				{licenseUtilizationPercentage && (
+					<div css={{ marginBottom: 24, height: 200 }}>
+						<ChartSection title="License Utilization">
+							<LinearProgress
+								variant="determinate"
+								value={licenseUtilizationPercentage * 100}
+								css={{
+									height: 24,
+									borderRadius: 4,
+									marginBottom: 8,
+								}}
+							/>
+							<span
+								css={{
+									fontSize: "0.75rem",
+									display: "block",
+									textAlign: "right",
+								}}
+							>
+								{Math.round(licenseUtilizationPercentage * 100)}% used (
+								{entitlements!.features.user_limit.actual}/
+								{entitlements!.features.user_limit.limit} users)
```

Comment:
Would it make sense to give these a similar treatment to `licenseUtilizationPercentage` and check if we can evaluate them based on whether `entitlements` is nil?